### ISSUE TITLE
CSS Problem

### ISSUE THREAD

**Issue by guissmo** [PhieF/CarnetNextcloud#17]
Hi, I'm having a CSS problem with Carnet.

![screenshot from 2018-10-28 14-58-06](https://user-images.githubusercontent.com/15175609/47616823-f93f0400-dac1-11e8-8965-68fb1a3bf33a.png)

**Comment by PhieF** [comment]
Hi

how did you install nextcloud ? docker ? Snap ? 
Thanks for your report, are you using at least 0.8.9 version of Carnet ?

**Comment by guissmo** [comment]
The CSS works when I have https://domain.com/cloud/index.php/XXX but it doesn't work (i.e. looks like the screenshot) when I visit https://cloud.domain.com/XXX. I installed it manually.

Here are some lines in my config that might help.
  'overwrite.cli.url' => 'https://domain.com/cloud/',
  'htaccess.RewriteBase' => '/',

Thanks for the reply!

**Comment by AaronADev** [comment]
@guissmo So doesn't work when pretty urls are enabled?

Did you change your config like this?

```
'htaccess.RewriteBase' => '/',
```

to get rid of the index.php? Just curious, I didn't have the same issue. Phief is pretty good at getting stuff resolved, I'm sure he'll figure it out.

**Comment by guissmo** [comment]
> @guissmo So doesn't work when pretty urls are enabled?
> 
> Did you change your config like this?
> 
> ```
> 'htaccess.RewriteBase' => '/',
> ```
> 
> to get rid of the index.php? Just curious, I didn't have the same issue. Phief is pretty good at getting stuff resolved, I'm sure he'll figure it out.

Yeah. I have a feeling I misconfigured something somehow. As I said above, I do have htaccess.RewriteBase set to '/', but the URL that works still has "index.php" for some reason. If I remove "index.php", it just says Forbidden.

Whenever I update the config.php, do I have to do something? I run sudo -u www-data php occ maintenance:update:htaccess every time I do that.

I'm quite new here but I hope I managed to make the situation clear.

**Comment by guissmo** [comment]
I reinstalled the SSL certificate and it got fixed. My bad.
